bench/BINARY_SIZE.md

```diff
@@ -16,6 +16,18 @@ The programs and their tests are located in [/tests/bench](https://github.com/co
 
 Solana version: 2.1.0
 
+| Program | Binary Size | -                      |
+| ------- | ----------- | ---------------------- |
+| bench   | 1,127,424   | 🔴 **+85,496 (8.21%)** |
```

Comment:
Having a binary size change in a version bump PR doesn't make sense and should never happen. CI would have failed in previous PRs if this was the case. Make sure your active Solana tooling version matches the "Solana version".

If you want to change the Solana version, that should ideally be done in a separate PR (e.g. https://github.com/solana-foundation/anchor/pull/3339) because upgrading Solana tooling version might require additional code changes, which would go against the nature of our own version bump PRs.

Reply:
Yeah looks like there was a regression too. We'll have to do a separate pr for the update then this one